lib/api/lib.api

```diff
@@ -5,6 +5,7 @@ public final class com/google/android/material/composethemeadapter/MdcTheme {
 }
 
 public final class com/google/android/material/composethemeadapter/ThemeParameters {
+	public static final field $stable I
```

Comment:
What is this BTW?

Reply:
I think it's part of the new Compose inference stuff, but not sure.